src/models/client.ts

```diff
@@ -209,30 +219,65 @@ export class ClientEnvironment {
     await this.network.clearBrowserCache();
   }
 
-  public async navigate(arg?: string | NavigateOptions): Promise<NavigateResult> {
-    const targetUrl = arg ? typeof arg === 'string' ? arg : arg.targetUrl : null;
-    const url = targetUrl ? this.getAbsoluteUrl(targetUrl) : this.rootUrl;
-    const waitForLoad = typeof arg === 'object' && arg.waitForLoad;
-
-    const tree = await this.page.getFrameTree();
-    const frameId = tree.frameTree.frame.id;
+  private async buildCapturedResponse(response: Network.ResponseReceivedParameters, shouldAddBodies: boolean) {
+    const capturedResponse = {responseMeta: response.response} as CapturedResponse;
 
-    const navPromise = this.frameStore.start(frameId, waitForLoad);
-    await this.page.navigate({ url });
+    if (shouldAddBodies) {
```

Comment:
should `shouldAddBodies` be renamed to `shouldAddBody`, since there appears to only be one body?

Reply:
Yeah, I've renamed. It was a carry over from the var name in its parent method.